infrastructure/dogfood/terraform/aws/vuln_processing.tf

```diff
@@ -0,0 +1,151 @@
+resource "aws_ecs_task_definition" "vuln-processing" {
```

Comment:
Should we be modifying old dogfood code or just leaving it deprecated?

Reply:
I think its safe to add since its disabled by default, and adds an example of how to do it for folks that are still using the legacy terraform.